content/en/blog/2022/egress-policies/index.md

```diff
@@ -0,0 +1,1105 @@
+
+![Egress traffic](egress-external.png)
+
+{{< tip >}}
+Is important to note that for this example relies on Istio's automatic mutual TLS, this means services within the mesh send TLS traffic and we are only sending `SIMPLE` TLS traffic at the egress when requests leave the mesh to the actual external host. For `mTLS` origination for egress traffic the `DestinationRule` needs to define the secret name that holds the client credentials certificate and be on `MUTUAL` mode. See more details [here](/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/#configure-mutual-tls-origination-for-egress-traffic).
+{{< /tip >}}
+
+***
+
+### Route internal outbound traffic to the egress gateway
+
+After deleting the `ServiceEntry`s used on the previous section, make sure your mesh is still blocking outbound access, and that there are no other resources that can conflict with the configuration like other `DestinationRule`s, `VirtualService`s, `Gateway`s and `AuthorizationPolicy`:
+
+{{< text bash >}}
+$ kubectl exec $SLEEP_POD1 -it -- curl -I https://developers.google.com
+$ kubectl exec $SLEEP_POD2 -n otherns -it -- curl -I https://developers.google.com
+$ kubectl exec $SLEEP_POD1 -it -- curl -I https://developer.yahoo.com
+$ kubectl exec $SLEEP_POD2 -n otherns -it -- curl -I https://developer.yahoo.com
+{{< /text >}}
+
+For all requests expect an error along the lines:
+
+{{< text plain >}}
+curl: (35) OpenSSL SSL_connect: Connection reset by peer in connection to developer.yahoo.com:443
+command terminated with exit code 35
+{{< /text >}}
+
+Analyze the following resources `external-google.yaml` and `external-yahoo.yaml`:
+
+Google:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: google
+spec:
+  hosts:
+  - developers.google.com
+  ports:
+  - number: 80
+    name: http
+    protocol: HTTP
+  - number: 443
+    name: https
+    protocol: HTTPS
+  resolution: DNS
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-google-egressgateway
+spec:
+  selector:
+    istio: egressgateway # use Istio default gateway implementation
+  servers:
+  - port:
+      number: 80
+      name: https-port-for-tls-origination
+      protocol: HTTPS
+    hosts:
+    - developers.google.com
+    tls:
+      mode: ISTIO_MUTUAL
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: egressgateway-for-google
+spec:
+  host: istio-egressgateway.istio-system.svc.cluster.local
+  subsets:
+  - name: google
+    trafficPolicy:
+      portLevelSettings:
+      - port:
+          number: 80
+        tls:
+          mode: ISTIO_MUTUAL
+          sni: developers.google.com
+---
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
I know our existing examples do this, but IMO we should have 2 VSs. It is way more understandable and allows putting one in app namespace and one in egress ns (if desired).

I would be appreciative if all our examples were updated to this split model as well, although certainly not a block for this PR

Reply:
This is fine for now. But just to be clear - just mean splitting the 1 VS into 2 VS. They can even be in the same namespace if we really want

Reply:
was there another step to be done here?

Reply:
@howardjohn ^